package.json

```diff
@@ -184,5 +184,8 @@
       "node14-win-x64"
     ],
     "outputPath": "dist"
+  },
+  "volta": {
+    "node": "16.14.2"
```

Comment:
- Is this intended ? Monika contribution guide does not explain Volta as its standard environment
- Node v16 does not reflect `pkg`'s targets

Reply:
yes, node v16.xx is the recommended target build env.